packages/backend/src/config/app.ts

```diff
@@ -92,3 +99,27 @@ export const Config = {
   /** Frontend **/
   frontendUrl: envString('FRONTEND_URL', 'http://localhost:3000')
 }
+
+function parseRedisTlsConfig(
+  caFile: string,
+  keyFile: string,
+  certFile: string
+): ConnectionOptions | undefined {
+  if (caFile === '' && keyFile === '' && certFile === '') return undefined
+
+  const options: ConnectionOptions = {}
+
+  // self-signed certs.
+  if (caFile !== '') {
+    options.ca = readFileSync(caFile)
+    options.rejectUnauthorized = false
+  }
+
+  // Both are required for client authentication.
+  if (keyFile !== '' || certFile !== '') {
+    options.key = readFileSync(keyFile)
+    options.cert = readFileSync(certFile)
+  }
+
+  return options
```

Comment:
```suggestion
  return Object.keys(options).length ? options : undefined
```
If only `keyFile` or `certFile` are specified, `options` would be empty.
If this check ends up being needed (depending on how only one of `key`/`cert` is handled) the check at the top of the function could be removed.